### PR TITLE
Support new templates for oncall integration

### PR DESCRIPTION
**Pull request by Konstantinov-Innokentii**
This PR adds support of web, telegram, msteams, phone_call, sms, acknowkedge_signal and source_link integration templates for oncall_integration resource.

### CODE REVIEW

**Review by inkel — COMMENTED** [review]
First pass LGTM. I'm going to give it another review later on.

File: internal/resources/oncall/resource_integration.go [hunk 1]
```diff
@@ -205,6 +215,131 @@ func ResourceIntegration() *schema.Resource {
...
+										Description: "Template for Alert image url.",
+									},
+								},
+							},
+							MaxItems: 1,
+						},
+						"microsoft_teams": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							Description: "Templates for Micrisoft Teams.",
```
Comment (julienduchesne) [inline internal/resources/oncall/resource_integration.go]:
```suggestion
							Description: "Templates for Microsoft Teams.",
```

File: internal/resources/oncall/resource_integration.go [hunk 2]
```diff
@@ -205,6 +215,131 @@ func ResourceIntegration() *schema.Resource {
...
+									},
+									"image_url": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Description: "Template for Alert image url.",
+									},
+								},
+							},
+							MaxItems: 1,
+						},
```
Comment (julienduchesne) [inline internal/resources/oncall/resource_integration.go]: These blocks are all very similar, they could be generated by a single function
```suggestion
func onCallTemplate(name string, hasMessage, hasImage bool) *schema.Schema
...
						"telegram": onCallTemplate("Telegram", true, true),
```

**Review by julienduchesne — APPROVED** [review]
Nice! LGTM!

**Pull request merged**